<ui version="4.0" >
 <class>CheatCheatDialog</class>
 <widget class="QDialog" name="CheatCheatDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>276</width>
    <height>439</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Cheat</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2" >
   <item row="0" column="0" >
    <layout class="QGridLayout" name="gridLayout" >
     <item row="0" column="0" >
      <widget class="QLabel" name="labelName" >
       <property name="text" >
        <string>Name:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" >
      <widget class="QLineEdit" name="lineEditName" >
       <property name="maximumSize" >
        <size>
         <width>16777215</width>
         <height>23</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="2" >
      <widget class="QRadioButton" name="radioDisabled" >
       <property name="maximumSize" >
        <size>
         <width>16777215</width>
         <height>23</height>
        </size>
       </property>
       <property name="text" >
        <string>Disabled</string>
       </property>
       <property name="checked" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="0" colspan="2" >
      <widget class="QRadioButton" name="radioEnabled" >
       <property name="maximumSize" >
        <size>
         <width>16777215</width>
         <height>23</height>
        </size>
       </property>
       <property name="text" >
        <string>Enabled (Session)</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0" colspan="2" >
      <widget class="QRadioButton" name="radioAlwayEnabled" >
       <property name="minimumSize" >
        <size>
         <width>0</width>
         <height>22</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>16777215</width>
         <height>23</height>
        </size>
       </property>
       <property name="text" >
        <string>Enabled (Always)</string>
       </property>
      </widget>
     </item>
     <item row="15" column="0" colspan="3" >
      <widget class="QDialogButtonBox" name="buttonBox" >
       <property name="maximumSize" >
        <size>
         <width>16777215</width>
         <height>29</height>
        </size>
       </property>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons" >
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
     <item row="14" column="0" colspan="2" >
      <widget class="QTextEdit" name="textEditComment" >
       <property name="enabled" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="13" column="0" >
      <widget class="QLabel" name="labelComment" >
       <property name="text" >
        <string>Comment:</string>
       </property>
      </widget>
     </item>
     <item rowspan="3" row="9" column="1" >
      <widget class="QTableView" name="tableView" >
       <property name="showGrid" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="9" column="0" >
      <widget class="QPushButton" name="pushAdd" >
       <property name="text" >
        <string>&amp;Add</string>
       </property>
      </widget>
     </item>
     <item row="10" column="0" >
      <widget class="QPushButton" name="pushRemove" >
       <property name="text" >
        <string>&amp;Remove</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>CheatCheatDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel" >
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>CheatCheatDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel" >
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
